README.md

```diff
@@ -1 +1,45 @@
-# DreamHome_BEE
\ No newline at end of file
+# DreamHome BackEnd
+
+
+## About
+
+## Setup Instructions
+  - how to run the dev/test server locally
+  - how to run production deployed server
+
+## Visual Schema
+
+## Tech stack
+
+## Development Database Instructions
+
+## Endpoint Documentation
+
+## File Tree
```

Comment:
I like that this is here for now, once we complete the MVP server we can update this to an image